docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>At the end of this, consider adding a screenshot of TensorBoard.dev as seen in the browser.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>